amy/extforms/tests/test_training_request_form.py

```diff
@@ -62,6 +62,30 @@ def setUpMembership(self):
             agreement_end=date.today() + timedelta(weeks=26),
             contribution_type="financial",
             registration_code="valid123",
+            public_instructor_training_seats="1",
+            inhouse_instructor_training_seats="1",
```

Comment:
Minor: is this a string value?

Reply:
Nope, they are ints, not sure why I put them in as strings